samples/ControlCatalog/MainWindow.xaml

```diff
@@ -45,10 +45,10 @@
   <DockPanel LastChildFill="True">
     <Menu Name="MainMenu" DockPanel.Dock="Top">
       <MenuItem Header="File">
-        <MenuItem Header="Exit" Command="{Binding ExitCommand}" />
+        <MenuItem Header="Exit" Command="{Binding Exit}" />
```

Comment:
prefer to keep unrelated changes in seperate PR.

I think this isnt releated to notifications?

Reply:
It is not. I just thought that it makes it more readable.

Reply:
I understand, but we will have to revert this change :) thanks